docs/components/dartpad/sliders/discrete_slider/main.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatefulWidget {
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  double _sliderDiscreteValue = 20;
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: Column(
+          children: [
+            Expanded(
+              child: Slider(
+                value: _sliderDiscreteValue,
+                min: 0,
+                max: 100,
+                divisions: 5,
+                label: _sliderDiscreteValue.round().toString(),
+                onChanged: (value) {
+                  setState(() {
+                    _sliderDiscreteValue = value;
+                  });
+                },
+              ),
+            ),
+            Expanded(
+              child: SliderTheme(
+                data: SliderThemeData(
```

Comment:
Why is there 2 sliders in this demo? And isn't the paddle shape deprecated?

Reply:
Done